pallets/msa/src/benchmarking.rs

```diff
@@ -211,8 +211,14 @@ benchmarks! {
 			let sig = generate_test_signature();
 			assert_ok!(Msa::<T>::register_signature(&sig, T::BlockNumber::from(mortality_block)));
 		}
+
```

Comment:
We might have to do some rethinking here.

WEIGHT_PER_SECOND = 1_000_000_000_000 = 10^12

We have two seconds of computation time:
- 2 * 10^12

Cost of 1 write:
1 write = 100_000 * constants::WEIGHT_PER_NANOS = 100_000 * 1_000 = 10 ^ 8

Msa configuration:
- MSAMaxSignaturesPerBucket = 50_000

approx on_initialize weights:
```
 Weight::from_ref_time(0 as u64)
        // Standard Error: 5_000
        .saturating_add(Weight::from_ref_time(1_242_000 as u64).saturating_mul(m as u64))
        .saturating_add(T::DbWeight::get().writes((1 as u64).saturating_mul(m as u64)))
```
an extreme case, if we had 50_000 signatures
10^8 * 50_000 = 5_000_000_000_000 

This exceeds the block space by more than two times without including any other operational-related weights.

There are few solutions to this posted on [stackexchange](https://substrate.stackexchange.com/questions/5842/on-initialize-best-practice-for-executing-a-large-for-loop)
- queue
- using on_idle to slowly drain on each block...ect